src/redux/reducers/usersReducer.js

```diff
@@ -0,0 +1,46 @@
+import { initialState } from '../../utils/initialState';
+
+import { FOLLOW, UNFOLLOW, SET_USERS } from '../actions/actionTypes';
+
+const usersReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FOLLOW:
+      return {
+        ...state,
+        users: state.users.map((user) => {
+          if (user.id === action.userID) {
+            return { ...user, followed: true };
+          }
+          return user;
+        }),
+      };
+
+    case UNFOLLOW:
+      return {
+        ...state,
+        users: state.users.map((user) => {
+          if (user.id === action.userID) {
+            return { ...user, followed: false };
+          }
+          return user;
+        }),
+      };
+
+    case SET_USERS:
+      return {
+        ...state,
+        users: [...state.users, ...action.users],
+      };
+
+    default:
+      return state;
+  }
+};
+
+const followActionCreator = (userID) => ({ type: FOLLOW, userID });
```

Comment:
Consider moving these function to "actionCreators".

Reply:
/fxd